src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+// ACTIONS
+const ADD = 'add_book';
+const REMOVE = 'remove-book';
```

Comment:
- [ ] In keeping with the ducks pattern `(app/reducer/ACTION_TYPE)`, kindly rename your action types here: For example `const ADD = bookstore/books/ADD`.

- [ ] **[OPTIONAL]** Wouldn't it be better to rename the action types to something more descriptive and apparent like `ADD_BOOK` and `REMOVE_BOOK`? In the end you can have something like so: `const ADD_BOOK = bookstore/books/ADD_BOOK`